### PR TITLE
162320674 show all data button graph

### PR DESCRIPTION
**Pull request by mklewandowski**
Add a button to the pop sim graph that allows the user to toggle between all data points and recent data points.

### CODE REVIEW

File: src/components/two-up-display.tsx [hunk 1]
```diff
@@ -90,11 +92,41 @@ export class TwoUpDisplayComponent extends BaseComponent<IProps, IState> {
...
           {this.props.rightPanel}
         </div>
-        {/* <div className={"footer " + this.props.spaceClass} data-test="right-footer"/> // removed for time being*/}
+        {this.renderRightPanelFooter()}
       </div>
     );
   }
 
+  private renderRightPanelFooter(){
+    if (this.props.spaceClass === "populations" &&
```
Comment (mklewandowski) [inline src/components/two-up-display.tsx]: Not sure if the button is going to eventually live in the footer, but we will need this space for other graph buttons eventually.  This button might end up floating on top of the graph in the upper right corner.

File: src/components/two-up-display.tsx [hunk 2]
```diff
@@ -90,11 +92,41 @@ export class TwoUpDisplayComponent extends BaseComponent<IProps, IState> {
...
+
+      return (
+        <div className={"footer " + this.props.spaceClass} data-test="right-footer">
+          { buttons }
+        </div>
+      );
+    } else {
+      return null;
+    }
+
```
Comment (mklewandowski) [inline src/components/two-up-display.tsx]: Not sure if the button is going to eventually live in the footer, but we will need this space for other graph buttons eventually.  This button might end up floating on top of the graph in the upper right corner.

File: src/models/spaces/populations/mouse-model/mouse-populations-model.ts [hunk 3]
```diff
@@ -195,6 +219,19 @@ export const MousePopulationsModel = types
...
+          const buttons = [];
+
+          buttons.push({
+            title: "Scale",
+            value: self.showMaxPoints,
+            action: (val: boolean) => {
+              self.toggleShowMaxPoints();
+            }
+          });
+
```
Comment (mklewandowski) [inline src/models/spaces/populations/mouse-model/mouse-populations-model.ts]: I liked this idea of storing the toolbar buttons in an array model so I recycled it here.

Comment (sfentress) [reply src/models/spaces/populations/mouse-model/mouse-populations-model.ts]: :+1: This worked well when I copied the pattern for the other buttons.

**Comment by mklewandowski** [comment]
I don't have a strong opinion.  My gut reaction was to put everything in the footer for now and then restructure things if and when we have final styling and a prioritized story telling us to take care of it.  What do you think?

**Comment by mklewandowski** [comment]
Looking at the v.2.3 UI spec now.  There is still nothing specified other than the styling of buttons in the footer (which is the styling I used for this button).  I think we should just put everything there for now.

**Comment by sfentress** [comment]
👍

**Pull request merged**